pyhydroquebec/__main__.py

```diff
@@ -96,22 +100,42 @@ def main():
         print(VERSION)
         return 0
 
-    if not args.username or not args.password:
+    # Check input for Username, Password and Contract - CLI overwrite ENV variable
+    hydro_user = hydro_pass = hydro_contract = None
+    #Check Env
+    if "PYHQ_USER" in os.environ:
+        hydro_user = os.environ["PYHQ_USER"]
+
+    if "PYHQ_PASSWORD" in os.environ:
+        hydro_pass = os.environ["PYHQ_PASSWORD"]
+
+    if "PYHQ_CONTRACT" in os.environ:
+        hydro_contract = os.environ["PYHQ_CONTRACT"]
+
+    #Check Cli
+    if args.username:
+        hydro_user = args.username
+    if args.password:
+        hydro_pass = args.password
+    if args.contract:
+        hydro_contract = args.contract
+
+    if not hydro_user or not hydro_pass:
         parser.print_usage()
         print("pyhydroquebec: error: the following arguments are required: "
-              "-u/--username, -p/--password")
+                "-u/--username, -p/--password")
```

Comment:
Keep it aligned.
The `tox -e pylint,lint` command should highlight this issue

Reply:
Done